daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/UnseparatedSequenceUnparsers.scala

```diff
@@ -43,8 +43,6 @@ class RepOrderedUnseparatedSequenceChildUnparser(
   extends RepeatingChildUnparser(childUnparser, srd, erd)
   with Unseparated {
 
-  private val ock = erd.maybeOccursCountKind.get
-
```

Comment:
I don't see an override here.  Was this a redundant definition of ock that Scala didn't warn about or what?

Reply:
The one from MinMaxRepeatsMixin was private, so this couldn't use the one from the mixin and so had to redefine it. If the one in MinMaxRepeatsMixin was protected/public, I imagine scala would have created a warning. But since they were the same, we can just use the definition from MinMaxRepeatsMixin.